content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,121 @@
+# Requirements
+
+When you have a model saved via MLEM and want to use it, the first step is to
+make sure you have the right dependencies and packages. MLEM can get the list of
+requirements for you automatically for some type of requirements.
+
+## Pip based requirements
+
+`MLEM` can export the installable requirements needed for a model using the
+
+`mlem build requirements -m model`
```

Comment:
Just `mlem build`, since you're printing the exact command below and since the docs page you linking is general for `build` - it don't have any special info about `requirements` subcommand. And no need in newlines :)

Reply:
done